### PR TITLE
Updated SecuredController to use non-deprecated classes whenever possible

### PR DESCRIPTION
**Pull request by Restless-ET**

### CODE REVIEW

File: Resources/skeleton/acme-demo-bundle/Acme/DemoBundle/Controller/SecuredController.php [hunk 1]
```diff
@@ -20,15 +21,24 @@ class SecuredController extends Controller
...
+        } else {
+            $error = $request->getSession()->get($authErrorKey);
         }
 
         return array(
-            'last_username' => $request->getSession()->get(SecurityContext::LAST_USERNAME),
-            'error'         => $error,
+            'last_username' => $request->getSession()->get($lastUsernameKey),
+            'error' => $error,
         );
```
Comment (fabpot) [inline Resources/skeleton/acme-demo-bundle/Acme/DemoBundle/Controller/SecuredController.php]: You can simplify that even more as of 2.7. See http://symfony.com/doc/current/cookbook/security/form_login_setup.html

Comment (Restless-ET) [inline Resources/skeleton/acme-demo-bundle/Acme/DemoBundle/Controller/SecuredController.php]: @fabpot I believe you're talking about using the 'security.authentication_utils', correct?
I can change this to use it, but I'll still need to verify it to keep BC for SF < 2.6 or am I missing something?

**Comment by Restless-ET** [comment]
@fabpot I've kept the code for BC with symfony versions < 2.6. I don't know if that's still required... if not I can remove it.

**Comment by fabpot** [comment]
Thank you @Restless-ET.

**Pull request closed without merging**